### ISSUE TITLE
Not getting Chirr

### ISSUE THREAD

**Issue by An-odd-person** [TeamMoonstorm/Starstorm2#671]
I’ve killed Empyrean Elites multiple times during my runs and haven’t unlocked Chirr. The first time I did it, I had guillotines, so I thought if I removed them, that’s fix it, yet on a separate run where I had no guillotines, it still didn’t work. I had the Artifact of Exponentiality (modded) and Artifact of Command on, so that might be it. I really want to unlock Chirr since I don’t want part of the mod to be locked out because of a bug.

**Comment by gamrtiem** [comment]
yeah from what ive seen the unlock seems to be broken in general for now ,,,, you can use [realercheatunlocks](https://thunderstore.io/package/prodzpod/RealerCheatUnlocks/) to unlock her for now ,,..

**Comment by An-odd-person** [comment]
Thank you

**Comment by Hevia** [comment]
should have been fixed in one of the updates from this year iirc
